tutorials/backup-postgresql-pgbackrest-s3/index.mdx

```diff
@@ -7,7 +7,8 @@ content:
   paragraph: This page shows how to backup PostgreSQL databases with pgBackRest to Object Storage
 tags: compute networking database sql postgresql pgbackrest backup buckets Object-Storage s3
 categories:
-  - storage
+  - object-storage
```

Comment:
If this one has the "object-storage" tag, should it also have the "storage" tag?